base/cvd/cuttlefish/host/commands/cvd/cli/selector/creation_analyzer.cpp

```diff
@@ -157,8 +157,14 @@ CreationAnalyzer::AnalyzeInstanceIdsInternal(
     }
     return instance_info;
   }
+
+  const auto incremental = selector_options_parser_.UseCvdalloc();
+  std::set<int> requested(requested_instance_ids.begin(),
+                          requested_instance_ids.end());
   auto acquired_all_file_locks =
-      CF_EXPECT(instance_lock_file_manager_.LockAllAvailable());
+      incremental
```

Comment:
Can you make it always acquire locks incrementally? regardless of whether cvdalloc should be used or not.